restler/restler_settings.py

```diff
@@ -441,6 +445,8 @@ def convert_wildcards_to_regex(str_value):
         self._ignore_feedback = SettingsArg('ignore_feedback', bool, False, user_args)
         ## Include user agent in requests sent
         self._include_user_agent = SettingsArg('include_user_agent', bool, True, user_args)
+        ## Include a unique sequence ID in requests sent
```

Comment:
For my own understanding, if this were set to false, would it be possible to determine which requests in the structured log were for the same sequence? (It looks like requests/responses could be linked using `combination_id`? But I wanted to confirm if that is correct).

Reply:
Yes, it is still possible.  This setting is independent of anything in the structured log, and applies to the payloads themselves.  I should probably split this off to its own PR which will go in after this one.  The same 'sequence_id' is also available in the structured log by default, in the tags.  'combination_id' is the grammar-defined combination, so it may appear twice if you are running in random mode, but the 'sequence_id' is a unique guid generated for every sequence run.